src/app/utils/globals.js

```diff
@@ -35,12 +37,15 @@ GLOBALS.DEFAULT_WAIFU_IMAGE =
 GLOBALS.WAIFUS_URL = "https://ipfs.io/ipfs/#"; //replace # with hash of the names.json (huge list of all waifus)
 GLOBALS.WAIFUS_JSON_HASH = "QmPUsgFPH6dFYZG2dPvRfE2mRrWSJ3wVLt2KZUNdE8vqDv";
 
-GLOBALS.WAIFUS_VIEWABLE_URL = "https://ipfs.io/ipfs/QmQuzMGqHxSXugCUvWQjDCDHWhiGB75usKbrZk6Ec6pFvw"; //SET THIS FOR WALLET PAGE TO LOAD WAIFU IMAGE
+GLOBALS.WAIFUS_VIEWABLE_URL = `https://ipfs.io/ipfs/${GLOBALS.WAIFU_VERSION === "eth" ? 'QmQuzMGqHxSXugCUvWQjDCDHWhiGB75usKbrZk6Ec6pFvw' : "QmT1X16PnT8jmx9StW2vMPhfMBoyr3ezE1a1uN7ekT3wBw"}`; //SET THIS FOR WALLET PAGE TO LOAD WAIFU IMAGE
 GLOBALS.GALLERY_VIEWABLE_URL =
-  "https://ipfs.io/ipfs/QmQuzMGqHxSXugCUvWQjDCDHWhiGB75usKbrZk6Ec6pFvw";
+  `https://ipfs.io/ipfs/${GLOBALS.WAIFU_VERSION === "eth" ? 'QmQuzMGqHxSXugCUvWQjDCDHWhiGB75usKbrZk6Ec6pFvw' : "QmT1X16PnT8jmx9StW2vMPhfMBoyr3ezE1a1uN7ekT3wBw"}`;

```

Comment:
any difference between these two?

Reply:
No difference, site just uses these interchangeably for some reason. We can fix in a cleanup